code/components/gta-streaming-five/component.json

```diff
@@ -8,7 +8,8 @@
 		"gta:core",
 		"vfs:core",
 		"rage:formats",
-		"vendor:tbb"
+		"vendor:tbb",
+		"vendor:msgpack-c"
```

Comment:
Ideally add new lines somewhere in the middle of the array so that it doesn't lead to 2 lines being changed.

Reply:
`"vendor:tbb"
,
"vendor:msgpack-c"`
Something like that?

Reply:
Heh, no, I rather meant adding the new entry elsewhere so you don't have to add a `,`.

Reply:
Ah true, now I see a solution.